io/include/pcl/io/pcd_io.h

```diff
@@ -84,6 +84,32 @@ namespace pcl
         PCD_V7 = 1
       };
 
+      /** \brief Read a point cloud data header from a PCD-formatted, binary istream.
+        *
+        * Load only the meta information (number of points, their types, etc),
+        * and not the points themselves, from a given PCD stream. Useful for fast
+        * evaluation of the underlying data structure.
+        *
+        * \attention The PCD data is \b always stored in ROW major format! The
+        * read/write PCD methods will detect column major input and automatically convert it.
+        *
+        * \param[in] binary_istream a std::istream with openmode set to std::ios::binary.
+        * \param[out] cloud the resultant point cloud dataset (only the header will be filled)
```

Comment:
No, the approach is alright in my opinion. It's just the docstring that should be changed. Something in the way "only the meta-information fields will be filled".